status.json

```diff
@@ -5934,5 +5934,23 @@
     "demo": "",
     "id": 6015941766807552,
     "uservoiceid": null
+  },
+  {
+    "name": "Background Sync API",
+    "category": "JavaScript",
+    "link": "https://wicg.github.io/BackgroundSync/spec/",
+    "summary": "Provides an API for notifying a Service Worker that the user has come back online, or to schedule periodic events to synchronize local data with the server.",
+    "standardStatus": "Working draft or equivalent",
+    "ieStatus": {
+      "text": "In Development",
+      "iePrefixed": "",
+      "ieUnprefixed": "",
+      "priority": "Low"
+    },
+    "spec": "bgsync",
+    "msdn": "",
+    "wpd": "",
+    "id": null,
```

Comment:
@nolanlawson Can you change this to: [`"id": 6170807885627392`](https://www.chromestatus.com/feature/6170807885627392)?